### PR TITLE
add operation and settings

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Operation record
+
+This topic will describe the operation record in Nebula Dashboard.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md:3]: the refers to specified nouns. the operation record -->operation records

Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md:3]: [describe the operation record] sounds weird. -->This topic shows how to use the operation record feature in xxx.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+![operation record](../figs/ds-018.png)
+
+On the **Operation Record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days or 14 days, including operator， operation， time and cluster name.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md:7]: record -->records

Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md:7]: [including operator， operation， time and cluster name.] sounds odd.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Other settings
+
+This topic will describe other settings in Nebula Dashboard.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:3]: --->The following shows other settings in xxx

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Information: It shows cluster name, creates time, and creator.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:5]: It --》it

Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:5]: please reorganize [cluster name, creates time, and creator.]

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- Unbind: Unbind a cluster and remove its information from the platform, doing nothing to the cluster's services and data.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:7]: , doing nothing to the cluster's services and data. --too oral
Data of Nebula Graph services will be retained.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+  !!! note
+
+        When unbinding a cluster, you need to enter the cluster name to unbind.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:11]: Before unbinding a cluster, enter the name of the cluster first.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+  ![Unbind](../figs/ds-019.png)
+
+- Delete: Delete a cluster and remove its information from the platform, stop the cluster's services and retain the data.
+
+  !!! note
+
+        When deleting a cluster, you need to enter the cluster name to delete.
```
Comment (abby-cyber) [inline docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:19]: ditto

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md (hunk 2 above)
Comment (izhuxiaoqing) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md:7]:
```suggestion
On the **Operation record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days, or 14 days. You can also view who runs what operation on which cluster at what time.
```

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md (hunk 4 above)
Comment (izhuxiaoqing) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:5]:
```suggestion
- Information: shows the cluster name, the creation time, and the creator.
```

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md (hunk 5 above)
Comment (izhuxiaoqing) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:7]: plz refer to the English page.

File: docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md (hunk 6 above)
Comment (izhuxiaoqing) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:11]: What abt "To unbind a cluster, enter the cluster name first"?

Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md:11]: > What abt "To unbind a cluster, enter the cluster name first"?

sure, perfect

**Pull request merged**